operator/src/main/java/com/github/streamshub/console/api/v1alpha1/Console.java

```diff
@@ -9,12 +9,17 @@
 import io.fabric8.kubernetes.client.CustomResource;
 import io.fabric8.kubernetes.model.annotation.Group;
 import io.fabric8.kubernetes.model.annotation.Version;
+import io.quarkiverse.operatorsdk.annotations.CSVMetadata;
 import io.sundr.builder.annotations.Buildable;
 
 @Version("v1alpha1")
 @Group("console.streamshub.github.com")
 @Buildable(editableEnabled = false, builderPackage = "io.fabric8.kubernetes.api.builder")
 @JsonInclude(JsonInclude.Include.NON_NULL)
+@CSVMetadata(
+    displayName = "Console Instance",
+    description = "Represents a deployable Console UI for monitoring Kafka resources."
+    )
```

Comment:
Does this really work for CRD metadata? I thought the CRDs were generated by Fabric8. I'm trying to understand in the JOSDK code how this works.